content/components/box.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Box
+react: https://primer.style/react/Box
+rails: https://primer.style/view-components/components/box
+description: Box is a basic wrapper component for most layout related needs.
+---
+
+import {Box, Text} from '@primer/react'
+
+## Usage
+
+The box component is considered a utility component, as it can be used for something as simple as a rounded corner box. 
+
+It's comprised of a container with a white background, a light gray border, and rounded corners. By default there are no additional styles, as these can be achieved using styled system props to enable custom theme-aware styling.
```

Comment:
I just double-checked this to be sure, but the `Box` component doesn't add any style by default (no background or border).

![Uploading Screenshot 2022-12-07 at 17.16.56.png…]()

Reply:
whoops, oversight here, I think this was taken from old box documentation